protostar/cairo_testing/cairo_injectable_function_runner.py

```diff
@@ -50,7 +50,8 @@ def run_cairo_function_by_offset(
             if self._cairo_runner_facade.did_panic():
                 panic_data = self._cairo_runner_facade.get_panic_data()
                 panic_data_short_strs = [
-                    short_string_to_str(datum) for datum in panic_data
+                    short_string_to_str(datum) if is_decodable_to_ascii(datum) else None
+                    for datum in panic_data
```

Comment:
I think we can hide this logic in `short_string_to_str` and just return `None` from it if datum is not decodable to ascii

Reply:
I think this function should fail / throw error whenever we try to convert something that is not a short string to a string. This function is also used in other places, and I believe that we want to know when the argument we are passing to the function is invalid.

Reply:
But the error reporting could be improved, I'll make a commit in a sec

Reply:
Could we reuse this logic in `protostar/cheatable_starknet/controllers/io.py` though? It would be more readable imo